acceptance-tests/features/steps/add_borrowers_definitions.rb

```diff
@@ -25,24 +20,19 @@
       }
     ]
   }
-  @response = HTTP.post(Env.domain + '/case/' + @created_case['id'].to_s +
+  @response = HTTP.post(Env.case_api + '/case/' + @created_case_id.to_s +
                         '/borrowers', json: borrower_json)
 end
 
 Then(/^the correct borrowers details are returned$/) do
-  number_of_borrowers = @added_borrowers['borrowers'].count
-
-  for i in 0..(number_of_borrowers - 1) do
-    added_borrower = @added_borrowers['borrowers'][i]
-    retrieved_borrower = @retrieved_borrowers[i]
-
-    assert_equal(added_borrower['first_name'], retrieved_borrower['first_name'])
-    assert_equal(added_borrower['middle_names'],
-                 retrieved_borrower['middle_names'])
-    assert_equal(added_borrower['last_name'], retrieved_borrower['last_name'])
-    assert_equal(added_borrower['address'], retrieved_borrower['address'])
-    assert_equal(added_borrower['email_address'],
-                 retrieved_borrower['email_address'])
-    assert_equal(added_borrower['mobile_no'], retrieved_borrower['mobile_no'])
+  @added_borrowers.zip(@retrieved_borrowers).each do |added, retrieved|
+    assert_equal(added['first_name'], retrieved['first_name'])
+    assert_equal(added['middle_names'],
+                 retrieved['middle_names'])
+    assert_equal(added['last_name'], retrieved['last_name'])
+    assert_equal(added['address'], retrieved['address'])
+    assert_equal(added['email_address'],
+                 retrieved['email_address'])
+    assert_equal(added['mobile_no'], retrieved['mobile_no'])
```

Comment:
Dont change it for this but in the future it would be nice if spacing was more consistent. All these lines are around the same length, some probably go over the threshold for line length and those have been spaced onto new lines but since all of the other lines are just around the same length it would make the code more readable to make them consistent e.g.

```
@added_borrowers.zip(@retrieved_borrowers).each do |added, retrieved|
  assert_equal(added['first_name'],
               retrieved['first_name'])
  assert_equal(added['middle_names'],
               retrieved['middle_names'])
  assert_equal(added['last_name'],
               retrieved['last_name'])
  assert_equal(added['address'],
               retrieved['address'])
  assert_equal(added['email_address'],
               retrieved['email_address'])
  assert_equal(added['mobile_no'],
               retrieved['mobile_no'])
```

This to me looks more readable. Worth discussing if you disagree